lib/graticule/geocoder/geonames.rb

```diff
@@ -3,12 +3,13 @@ module Graticule #:nodoc:
   module Geocoder #:nodoc:
     class Geonames < Base
 
-      def initialize
+      def initialize(username = nil)
```

Comment:
We're maintaining it, but not actively seeking bugs to fix that aren't affecting us (or being squeaky wheels).

Actually @bryckbost merged a version of this in October which makes the username required. I'm closing this.